src/client/components/MealList.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
+
+const MealList = () => {
+  const [meal, setMeal] = useState();
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    (async () => {
+      setIsLoading(true);
+      const resp = await fetch("/api/meals");
+      const result = await resp.json();
+      console.log(result);
+      setMeal(result);
+      setIsLoading(false);
+    })();
+  }, []);
+
+  return (
+    <div>
+      {meal ? (
+        <ul>
+          {meal.map((aMeal) => {
```

Comment:
```suggestion
          {meals.map((meal) => {
```

Reply:
Hi Karo,
thanks for the comment. I implemented the code.